www/tasks/javascript.js

```diff
@@ -27,11 +29,20 @@ gulp.task('javascript', ['jshint', 'jscs'], function() {
 			loadMaps: true
 		}))
 		// Add transformation tasks to the pipeline here.
-		.pipe(uglify())
-		.pipe(sourcemaps.write('./'))
+		.pipe(gulpif(release, uglify()))
```

Comment:
Just thought this would be a lot cleaner option, if we think it's not necessary I can remove this dependency and use gutil.noop()